torch/csrc/jit/python/init.cpp

```diff
@@ -1468,16 +1468,50 @@ void initJITBindings(PyObject* module) {
     return type.value();
   });
 
-  py::class_<FunctionSchema>(m, "FunctionSchema")
+    py::class_<Argument>(m, "Argument")
+      .def("__repr__", [](Argument& self) {
+        std::ostringstream s;
+        s << *self.type();
+        if (self.is_out()) {
+          s << "&";
+        }
+        s << " " << self.name();
+        return s.str();})
+      .def_property_readonly("name", [](Argument& self) { return self.name(); })
+      .def_property_readonly("type", [](Argument& self) { return self.type(); })
+      .def_property_readonly(
+          "N",
+          [](Argument& self) -> py::object {
+            return (self.N()) ? py::cast(*self.N()) : py::none();
+          })
+      .def_property_readonly(
+          "default_value",
+          [](Argument& self) -> py::object {
+            if (!self.default_value()) {
+              return py::none();
+            }
+            IValue v = *self.default_value();
+            return toPyObject(std::move(v));
+          })
+      .def(
+          "has_default_value",
+          [](Argument& self) -> py::bool_ {
+            return self.default_value().has_value();
+          })
+      .def_property_readonly("kwarg_only", [](Argument& self) -> bool {
+        return self.kwarg_only();})
+      .def_property_readonly(
+          "is_mutable", [](Argument& self) -> bool { return self.is_out(); });
```

Comment:
What does the "is_out" means? We have some inplace ops that are not out= variants.